### ISSUE TITLE
"Error: need a version of git whose diff-tree command has the --combined-all-paths option"

### ISSUE THREAD

**Issue by thebravoman** [newren/git-filter-repo#94]
This is what I am doing 

```bash
$ git filter-repo --path app/themes/ --to-subdirectory-filter app/themes2
```

and I receive

```bash
Error: need a version of git whose diff-tree command has the --combined-all-paths option
```

The error is load and clear. It makes sense. Yet it is not helpful in any way. I am running git 2.20.1 and I can not find a way to understand if I should go up or down the git versions to get to "a version of git whose diff-tree commands..."

Then I see the Prerequisites that say

```
    git >= 2.22.0 at a minimum; some features require git >= 2.24.0
    python3 >= 3.5
```

I would suggest the message ""Error: need a version of git whose diff-tree command has the --combined-all-paths option" to be changed to ""Error: need a version of git whose diff-tree command has the --combined-all-paths option - at least 2.22.0 should do" or something of that sort.

**Comment by newren** [comment]
Yeah, the error made sense when git-2.22.0 didn't even exist yet, but could be simplified nowadays.

**Comment by newren** [comment]
Fixed in commit 15494bba8a (filter-repo: make git version requirement error message more direct, 2020-05-19).
